downstream/attributes/attributes.adoc

```diff
@@ -48,7 +48,8 @@
 :RunnerRpm: Ansible-runner rpm/container
 
 // Operators
-:OperatorPlatform: Ansible Automation Platform Operator
```

Comment:
I would be wary of removing this attribute unless you're certain it isn't used in other documents. You don't want to break other peoples books. You could leave the existing definition, or do a search and replace across the repo to make sure no issues result from the change.

Reply:
So I actually did a find and replace in VSC (and checked using `repo-search-show`) and replaced all references of the attribute earlier in the week, but I realized today I did it after you created the feature branch.

So I did it again just for these section to make it easier to view, assuming that when the feature branch rebases it will pick up all the other instances ? 

I am a bit afraid you're gonna have merge conflicts when you got to rebase? Would it make life easier if I did a full find and replace again on the feature branch? Or  just add back in the original and fix it post event 2?

Sorry for the long winded reply!

Reply:
Awesome! As long as you checked in main and replaced the attribute, we should be good. I'm going to have conflicts on the feature branch anyway; so not to worry.